Source/WebKit/NetworkProcess/webtransport/NetworkTransportSession.cpp

```diff
@@ -73,7 +73,7 @@ void NetworkTransportSession::sendStreamSendBytes(WebTransportStreamIdentifier i
 
 void NetworkTransportSession::streamSendBytes(WebTransportStreamIdentifier identifier, std::span<const uint8_t> bytes, bool withFin, CompletionHandler<void()>&& completionHandler)
 {
-    if (auto* stream = m_bidirectionalStreams.get(identifier))
+    if (RefPtr stream = m_bidirectionalStreams.get(identifier))
         stream->sendBytes(bytes, withFin);
     else if (auto* stream = m_sendStreams.get(identifier))
```

Comment:
This one cannot use RefPtr?

Reply:
That one uses `UniqueRef` as the value. I initially tried changing both & hit build errors.

Reply:
Seems we'll likely need a CheckedPtr then but doesn't have to be in this PR.

Reply:
yeah, we also need to add the support for CheckedRef/CheckedPtr in the static analyzers.